client/src/elm/Backend/PrenatalEncounter/Model.elm

```diff
@@ -194,7 +194,7 @@ type Msg
     | HandleSavedSocialHistory (WebData ())
     | SaveVitals PersonId (Maybe VitalsId) VitalsValue
     | HandleSavedVitals (WebData ())
-    | SavePrenatalPhoto PersonId (Maybe PrenatalPhotoId) PhotoUrl
+    | SavePrenatalPhoto PersonId (Maybe PrenatalPhotoId) ImageUrl
```

Comment:
same here, if we want to get rid of Photo, instead of Image, likely the function names should not be an exception.